java-dialogflow/.gitignore

```diff
@@ -0,0 +1,17 @@
+# Maven
+target/
+
+# Eclipse
+.classpath
+.project
+.settings
+
+# Intellij
+*.iml
+.idea/
+
+# python utilities
+*.pyc
+__pycache__
+
+.flattened-pom.xml
```

Comment:
FYI for the reviewer/ approver. The `.gitignore` file is added back. Issue #8683 has been created already